server/etc/httpd/conf.d/pulp_content.conf

```diff
@@ -0,0 +1,22 @@
+
+WSGISocketPrefix run/wsgi
+WSGIProcessGroup pulp-content
+WSGIApplicationGroup pulp-content
+WSGIScriptAlias /pulp/content /srv/pulp/content.wsgi
+WSGIDaemonProcess pulp-content user=apache group=apache processes=3 display-name=%{GROUP}
+WSGIImportScript /srv/pulp/content.wsgi process-group=pulp-content application-group=pulp-content
+
+<Files content.wsgi>
+    WSGIPassAuthorization On
+    WSGIProcessGroup pulp-content
+    WSGIApplicationGroup pulp-content
+    SSLRenegBufferSize  1048576
+    SSLVerifyDepth 9
```

Comment:
This is set to different values in different places, can they be all set to the same value?

```
$ git grep SSLVerifyDepth
nodes/parent/etc/httpd/conf.d/pulp_nodes.conf:  SSLVerifyDepth  5
nodes/parent/etc/httpd/conf.d/pulp_nodes.conf:  SSLVerifyDepth  5
server/etc/httpd/conf.d/pulp_apache_22.conf:    SSLVerifyDepth 3
server/etc/httpd/conf.d/pulp_apache_24.conf:    SSLVerifyDepth 3
server/etc/httpd/conf.d/pulp_content.conf:    SSLVerifyDepth 9
```